packages/firestore/src/remote/persistent_stream.ts

```diff
@@ -638,7 +636,6 @@ export class PersistentWriteStream extends PersistentStream<
   private handshakeComplete_ = false;
 
   constructor(
-    private databaseInfo: DatabaseInfo,
```

Comment:
FWIW, removing databaseInfo everywhere is probably creating a porting difference (web doesn't need it because of the way connection is plumbed through vs. the way grpc works on iOS / Android), but it's probably fine.